Dockerfile

```diff
@@ -135,12 +135,18 @@ ADD https://github.com/cartesi/machine-emulator-tools/releases/download/v${MACHI
 RUN dpkg -i /tmp/machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb \
   && rm /tmp/machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.deb
 
+# install built-ins
+ARG BUILTINS_VERSION=0.4.0
+ADD https://github.com/crypto-bug-hunters/builtins/releases/download/v${BUILTINS_VERSION}/builtins-${BUILTINS_VERSION}.tar.gz /tmp
+RUN tar -xf /tmp/builtins-${BUILTINS_VERSION}.tar.gz -C /usr/bin \
+    && rm /tmp/builtins-${BUILTINS_VERSION}.tar.gz
+
```

Comment:
If this is at `riscv64-build-stage` and `COPY` here, we save some space, since removing it at `RUN` won't save the space used at previous `ADD` layer.

Reply:
I see, I just copied how the machine emulator tools was being installed.

Reply:
I know, 

I think I do this the same wat elsewhere, but while reviewing it, I saw this as an opportunity to reduce the final image a bit.